autoload/gitmessenger/blame.vim

```diff
@@ -142,10 +142,12 @@ function! s:blame__after_blame(git) dict abort
 
     let author = matchstr(stdout[1], '^author \zs.\+')
     let author_email = matchstr(stdout[2], '^author-mail \zs\S\+')
+    let author_time = matchstr(stdout[3], '^author-time \zs\d\+')
     let self.contents = [
         \   '',
         \   ' History: #' . len(self.history),
         \   ' Commit: ' . hash,
+        \   ' Date: ' . strftime('%c', author_time),
```

Comment:
- `strftime` may not be available depending on system (Please see (`:help strftime`). Please check it exists as follows
- I prefer putting `Date:` after `Author:` because `git log` shows commits in the format
- Please convert string to number explicitly by `str2nr()`

```vim
let self.contents = [
    \   '',
    \   ' History: #' . len(self.history),
    \   ' Commit: ' . hash,
    \   ' Author: ' . author . ' ' . author_email,
    \ ]

if exists('*strftime')
    let self.contents += ['Date: ' . strftime('%c', str2nr(author_time))]
endif
```

Reply:
I think putting `Date:` after `Author:` and `Committer`(if exists) is better.
What do you think?